tests/tests/compiler/mappings/external_read_with_local_fail.leo

```diff
@@ -0,0 +1,30 @@
+/*
+namespace: Compile
+expectation: Fail
+*/
+
+program registry.aleo {
+    mapping users: address => bool;
+
+    transition register() {
+        return then finalize(self.caller);
+    }
+    finalize register(addr: address) {
+        Mapping::set(users, addr, true);
+    }
+}
+
+// --- Next Program --- //
+
+import registry.aleo;
+program relay.aleo {
+    mapping users: address => bool;
+
+    transition send(addr: address) {
+        return then finalize(addr);
+    }
+    finalize send(addr: address) {
+        let a:bool = Mapping::get(relay.aleo/users, addr);
```

Comment:
We might actually want this to pass in the future, since `relay.aleo/users` is a valid mapping.
Could you make an issue for this?